articles/Bitcoin-Big-Corporations.md

```diff
@@ -0,0 +1,55 @@
+|   |   |
+|---|---|
+| Title               | Could big corporations control Bitcoin? |
+| Author              | João Paulo Sant'Anna |
+| Featured text       | FIXME |
+| CT link             | FIXME |
+| CT date             | YYYY-MM-DD FIXME |
+| CT tweet            | FIXME |
+| YouTube link        | FIXME |
+| YouTube tweet       | FIXME |
+| Twitter full video  | FIXME |
+| Twitter clip 1      | FIXME |
+| Twitter clip 2      | FIXME |
+
+↑ Add more table rows for more social media links. See [this completed article](articles/simple-payment-verification.md) for a real example.
+
+Thesis statement - Will Bitcoin Succumb to Corporate Hegemony, or Will Decentralization Prevail?
+
+Pre-discussion, research and questions
+Title: Could big corporations control Bitcoin?
+
+Outline
```

Comment:
The outline looks good to me. Looking forward to reading the first draft 👍

Reply:
thanks a lot, will post it now